packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -915,6 +921,20 @@ export function TransferPanelMain({
                     forToken={selectedToken}
                     prefix={selectedToken ? 'BALANCE: ' : ''}
                   />
+                  {app.isDepositMode &&
+                    isTokenMainnetUSDC(selectedToken?.address) && (
+                      <TokenBalance
+                        balance={
+                          erc20L2Balances?.[CommonAddress.ArbitrumOne.USDC] ??
+                          constants.Zero
+                        }
+                        on={NetworkType.l2}
+                        forToken={Object.assign({}, selectedToken, {
+                          symbol: 'USDC'
+                        })}
```

Comment:
A bit more readable? (And we usually use spread operator over Object.assign in the codebase)
```suggestion
                        forToken={{
                          ...selectedToken,
                          symbol: 'USDC'
                        }}
```

Reply:
i did this because of type errors :'(
<img width="815" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/553af2d3-9268-4140-af0d-00230125b72c">